specification/storage/resource-manager/Microsoft.Storage/stable/2021-04-01/storage.json

```diff
@@ -4313,12 +4313,58 @@
         "filters": {
           "$ref": "#/definitions/BlobInventoryPolicyFilter",
           "description": "An object that defines the filter set."
```

Comment:
can we specify filter for container rule?
will the prefix match also work for container? (If so, please update prefix description)

Reply:
Filter cannot be specified for container, it can only be specified for blobs